lib/mock-nexmo.js

```diff
@@ -37,8 +52,16 @@ function MockNexmo(log, balanceThreshold) {
 
         log.info({ op: 'sms.send.mock' })
 
-        callback(null, {
-          messages: [{ status: '0' }]
+        // HACK: Enable remote tests to see what was sent
```

Comment:
Good point, I think it is ok to leave. We could make and issue and backlog for the `restsms` or something similar.